### DISCUSSION TITLE
Web GUI for Managing Rules

### DISCUSSION THREAD

**Discussion by SeaWolf-666** [Security-Onion-Solutions/securityonion#3800] [Unsupported Versions]
Hello,
Why is there not a web interface for managing rule tuning? Is there one and I am just missing it? It seems like something really basic and having to ssh in to modify rules seems like an unnecessary burden. 

If there is not a GUI for this in SO2, does anyone of one another tool that can be added?

**Comment by weslambert** [comment]
We don't currently provide a web interface for this, but it is on the roadmap.
